src/setup.py

```diff
@@ -44,7 +44,8 @@ def read(fname):
     ],
     install_requires=[
         'knack==0.1.1',
-        'msrest==0.4.26',
+        'msrest>=0.4.4',
```

Comment:
@Christina-Kang should comment here, AFAIK we need to keep these versions fixed.

Reply:
Yes, let's keep the versions fixed for now. Until Bharat checks in his change to our serializer.